supervisor/os/manager.py

```diff
@@ -366,11 +366,19 @@ async def update(self, version: AwesomeVersion | None = None) -> None:
     async def mark_healthy(self) -> None:
         """Set booted partition as good for rauc."""
         try:
-            response = await self.sys_dbus.rauc.mark(RaucState.GOOD, "booted")
+            responses = [
+                await self.sys_dbus.rauc.mark(RaucState.ACTIVE, "booted"),
+                await self.sys_dbus.rauc.mark(RaucState.GOOD, "booted"),
+            ]
```

Comment:
If order of operations doesn't matter we should use `asyncio.gather`:
```py
responses = await asyncio.gather(
    self.sys_dbus.rauc.mark(RaucState.ACTIVE, "booted"),
    self.sys_dbus.rauc.mark(RaucState.GOOD, "booted"),
)
```
However does order of operations matter? If we had a situation where the first succeeded and the other failed is there one we would prefer to be the one that succeeded? Also would we want to specifically log which failed? Because we aren't doing that right now.

Reply:
I had the same thought and my conclusion is that the order shouldn't matter - as it's not possible to do it atomically, marking it as active first sounds a bit more logical (so you don't end up with "good" but inactive slot) but in the end I don't expect only one of the operation would fail, unless there was some race condition. However, if we do as you suggest and one of the operations really fails for some reason, the other one still might be performed. Wouldn't it be better to do it sequentially as in the original code to get at least some consistency?

About logging, just in case one of the operation fails, the cause should be traceable from the logged exception, shouldn't it?

Reply:
Yea it sounds like there's one we would prefer be the one that succeeded in the case of a failure on the second so I agree then, I would leave the code as is and do it sequentially.

> About logging, just in case one of the operation fails, the cause should be traceable from the logged exception, shouldn't it?

Ah right, its `_LOGGER.exception` now instead of `error`. Ok makes sense then.